src/lib/api/work.fragments.js

```diff
@@ -617,8 +617,18 @@ export function pidToWorkId({ pid }) {
   };
 }
 
+const coverFragment = `fragment coverFragment on Manifestation {
+  cover {
+    detail
+    origin
+  }
+}`;
+
 const manifestationDetailsForAccessFactory = `fragment manifestationDetailsForAccessFactory on Manifestation {
   pid
+  ownerWork {
+    workId
```

Comment:
i think we need  a method to generate a link to material page .. maybe we should get title and creator also